docs/tutorials/src/13_saveload.md

```diff
@@ -0,0 +1,112 @@
+# Saveload
+
+`saveload` is a module that provides mechanisms to serialize and deserialize a `World`, it makes use of the popular [`serde`] library and requires the feature flag `serde` to be enabled for `specs` in your `Cargo.toml` file.
+
+At a high level, it works by defining an identifier `Component`, the `Marker`, as well as a `Resource`, the `MarkerAllocator`. Marked entities will be the only ones subject to serialization and deserialization.
+
+`saveload` also defines `SerializeComponents` and `DeserializeComponents`, these do the heavy lifting of exporting and importing, as such, they take a bunch of `SystemData` as parameters.
+
+Let's go over everything, point by point:
+
+## `Marker` and `MarkerAllocator`
+
+`Marker` and `MarkerAllocator<M: Marker>` are actually traits, a simple implementation is available with `U64Marker` and `U64MarkerAllocator`. While these work well, you may want use multiple `Marker` types (one for persistent storage, another for networked synchronization, ...), in which case, you could simply implement the traits over multiple [`newtype`]s or implement the trait once over a generic type. Let's see what the latter solution would look like in code:
```

Comment:
Existing example code seems to be using the other approach. Any reason you decided on the generic trait approach? Is the `id` used anywhere?

Reply:
I guess that I was distracted by my own idea, I do believe that it would be neat for U64Marker and U64MarkerAllocator to be generic over a 'marker' type but that's the subject of a different PR. I replaced the code with something a lot more simple for the sake of the tutorial